src/components/change-requests/review-change-request/review-change-request.test.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { render, screen } from "@testing-library/react";
+import { MemoryRouter, Route } from "react-router-dom";
+import ReviewChangeRequest from "./review-change-request";
+
+const renderComponent = (accepted: boolean) => {
+    return render(
+        <MemoryRouter initialEntries={['change-requests/37']}>
```

Comment:
I believe I wrote helper functions to avoid the need for putting a memory router in the test file. Check out this test file for an example: https://github.com/Northeastern-Electric-Racing/PM-Dashboard-v2/blob/1121297ac2263c7c55085c8faef3fd2a759f0ed7/src/components/change-requests/change-requests.test.tsx#L41